src/aat/resources/features/F-054/F-054.feature

```diff
@@ -0,0 +1,129 @@
+  Scenario: must return 200 and list of case data for the given case id
+    Given a successful call [to create a token for case creation as a citizen] as in [Citizen_Token_Creation_Data_For_Case_Creation]
+    And   another successful call [to create a full case as a citizen] as in [Citizen_Full_Case_Creation_Data]
+    And   a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains case data matching the case just created above]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a positive response is received
+    And   the response [code is HTTP-200]
+    And   the response has all other details as expected
+
+  @S-094 @Ignore # Response code mismatch, expected: 401, actual: 403 (defect RDM-6628)
+  Scenario:  must return 401 when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains an invalid user authorisation token]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-401]
+    And   the response has all other details as expected
+
+  @S-095
+  Scenario: must return 403 when request provides authentic credentials without authorized access to the operation
+    And   a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains an valid user authorisation token that does not have access to the operation]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-403]
+    And   the response has all other details as expected
+
+  @S-562 @Ignore # Response code mismatch, expected: 400, actual: 200 /RDM-7066
+  Scenario: must return negative response HTTP-400 when request contains a non-existing jurisdiction ID
+    Given a successful call [to create a token for case creation as a citizen] as in [Citizen_Token_Creation_Data_For_Case_Creation]
+    And   another successful call [to create a full case as a citizen] as in [Citizen_Full_Case_Creation_Data]
+    And   a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a non-existing jurisdiction ID]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-563 @Ignore # Response code mismatch, expected: 400, actual: 200 /RDM-7066
+  Scenario: must return negative response HTTP-400 when request contains a non-existing case type ID
```

Comment:
none existing ct id would be expected to return 404

Reply:
proceeding lines should contain a wording to relate to the initial case created.

Reply:
added Given a user with [an active profile in CCD] and changed to 404